scripts/deployfhirproxy.bash

```diff
@@ -202,11 +207,12 @@ fi
 #
 defsubscriptionId=$(az account show --query "id" --out json | sed 's/"//g') 
 
+echo "Checking Script Execution directory"
 # Test for correct directory path / destination 
-if [ -f "${script_dir}/$0" ] && [ -f "${script_dir}/fhirroles.json" ] ; then
-	echo "Checking Script execution directory..."
+if [ -f "${script_dir}/fhirroles.json" ] ; then
+	echo "  necessary files found, continuning..."
 else
-	echo "Please ensure you launch this script from within the ./scripts directory"
+	echo "  necessary files not found... Please ensure you launch this script from within the ./scripts directory"
```

Comment:
I believe $ echo with no options starts on a new line by default. Do we want the white space in the beginning of " necessary files...", or should we make it "Necessary files not found..."?

Reply:
the offset above simply indents the string so it reads as 
Please ensure... 
  necessary files ... 
echo with a /r will do what you are suggesting ;)